velox/docs/functions/spark/array.rst

```diff
@@ -0,0 +1,38 @@
+=============================
+Array Functions
+=============================
+
+.. js:function:: array_sort(array(E)) -> array(E)
+    :noindex:
+
+    Returns an array which has the sorted order of the input array(E). The elements of array(E) must
+    be orderable. Null elements will be placed at the end of the returned array. ::
+
+        SELECT array_sort(ARRAY [1, 2, 3]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [3, 2, 1]); -- [1, 2, 3]
+        SELECT array_sort(ARRAY [2, 1, NULL]; -- [1, 2, NULL]
+        SELECT array_sort(ARRAY [NULL, 1, NULL]); -- [1, NULL, NULL]
+        SELECT array_sort(ARRAY [NULL, 2, 1]); -- [1, 2, NULL]
+
+.. function:: in(x, element) -> boolean
+
+    Returns true if the array ``x`` equals to any value of ``element``. the elements and x must be same type.
```

Comment:
typos: 

```
Returns true if the array 'x' contains an element equal to 'element'. 
```

Drop "the elements and x must be same type."

Reply:
fixed it.